app/models/tslr_claim.rb

```diff
@@ -25,13 +15,24 @@ class TslrClaim < ApplicationRecord
     no_school: 2,
   }, _prefix: :employed_at
 
+  enum qts_award_year: {
+    "before_2013": 0,
+    "2013-2014": 1,
```

Comment:
How do you feel about using underscores instead of dashes so they can be symbols and the magic enum boolean methods are available?

Reply:
I don't have a strong opinion about it. My thinking was that we're probably not going to use those magic enum methods for eligible years and less code would need updating if they remained in this format. Happy to change it though

Reply:
My gut is telling me that underscores are probably safer if there isn't an explicit reason to prefer a dash.

Reply:
I've added a commit that replaces the dashes with underscores